### PR TITLE
Update developer guide

### PR DESCRIPTION
**Pull request by candyhy**
Add manual testing for list command in DG

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -637,14 +637,16 @@ testers are expected to do more *exploratory* testing.
 
    1. Double-click the jar file Expected: Shows the GUI with a set of sample properties and appointments. The window size may not be optimum.
 
-1. Saving window preferences
+
+2. Saving window preferences
```
Comment (dvdweien) [inline docs/DeveloperGuide.md:641]: wait i thought the 1. will automatically change to whatever number it is supposed to be?

Comment (candyhy) [reply docs/DeveloperGuide.md:641]: because I added a space between lines to reduce clutter, hence it doesn't

Comment (dvdweien) [reply docs/DeveloperGuide.md:641]: orhhh icic

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**